scylla/src/transport/node.rs

```diff
@@ -242,8 +242,7 @@ pub struct CloudEndpoint {
 
 /// Describes a database server known on Session startup, with already resolved address.
 #[derive(Debug, Clone)]
-#[non_exhaustive]
-pub struct ResolvedContactPoint {
+pub(crate) struct ResolvedContactPoint {
     pub address: SocketAddr,
     pub datacenter: Option<String>,
 }
```

Comment:
Let's not have any pub fields in non-pub structs.

Reply:
I wonder if there is a lint for that, let me check

Reply:
It looks like `unreachable_pub` should warn abut it. I wonder why it doesn't.